boards/views.py

```diff
@@ -33,13 +33,29 @@ class BoardNewView(FormView):
 
 @require_POST
 def create(req):
-    form = CategoryForm(req.POST)
+    form = CategoryForm(req.POST, req.FILES)
+    print("______________")
+    print(req.POST)
```

Comment:
```suggestion
    print(req.POST)
```
在測試可以留 `print()`，但推上來就可以拿掉了

Reply:
拍謝拍謝嗚嗚 手刀刪！